tests/ZiraLink.Api.Application.UnitTests/ZiraLink.Api.Application.UnitTests.csproj

```diff
@@ -10,6 +10,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="IdentityModel" Version="6.0.0" />
```

Comment:
I think you can remove this and reuse it

Reply:
Done.